__tests__/api.test.js

```diff
@@ -76,6 +76,16 @@ describe("/api/articles", () => {
 					expect(body.msg).toBe("Bad Request");
 				});
 		});
+		it.only("200: should respond with a property of comment_count", () => {
+			return request(app)
+				.get("/api/articles/12")
+				.expect(200)
+				.then(({ body }) => {
+					expect(body.article).toMatchObject({
+						comment_count: expect.any(Number),
```

Comment:
You shouldn't need to add this to the previous test. If you remove this line and it's not passing, you may need to use a more flexible matcher 👍

Reply:
Sorry, didn't realise I had a .only in there :facepalm: